spec/fixtures/form1010_ezr/veteran_prefill_data.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
I figured it was cleaner to move an expected dataset to a fixture. We can reuse this if we need to and it keeps the spec files from getting too bloated.